russian/sleep/brushing/brushing.md

```diff
@@ -0,0 +1 @@
+Вы идете в ванную почистить зубы. В стаканчике лежит две зубные счетки, красная и синяя, и вы застываете в размышлении и задаете себе вопрос: "Какую из зубных счеток выбрать?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.